.clang-tidy

```diff
@@ -1,28 +1,6 @@
 ---
-Checks:          '*'
+Checks:          '*,-fuchsia-*'
 WarningsAsErrors: '*'
-HeaderFilterRegex: '\/src\/'
-AnalyzeTemporaryDtors: false
-CheckOptions:    
-  - key:             google-readability-braces-around-statements.ShortStatementLines
-    value:           '1'
-  - key:             google-readability-function-size.StatementThreshold
-    value:           '800'
-  - key:             google-readability-namespace-comments.ShortNamespaceLines
-    value:           '10'
-  - key:             google-readability-namespace-comments.SpacesBeforeComments
-    value:           '2'
-  - key:             modernize-loop-convert.MaxCopySize
-    value:           '16'
-  - key:             modernize-loop-convert.MinConfidence
-    value:           reasonable
-  - key:             modernize-loop-convert.NamingStyle
-    value:           CamelCase
-  - key:             modernize-pass-by-value.IncludeStyle
-    value:           llvm
-  - key:             modernize-replace-auto-ptr.IncludeStyle
-    value:           llvm
-  - key:             modernize-use-nullptr.NullMacros
-    value:           'NULL'
+HeaderFilterRegex: '\/src\/.*'
```

Comment:
https://github.com/osmcode/libosmium/blob/7674775c11322368a40249eddbc3526eed661c73/.clang-tidy#L89 is my goto for best practices from @joto. I changed this to use `.*` because that clang-tidy does. I've not taken the time to understand the difference/implications between `HeaderFilterRegex: '\/src\/'` and `HeaderFilterRegex: '\/src\/.*'` but we should figure that out...